environment.yml

```diff
@@ -41,6 +41,8 @@ dependencies:
   - yamale
   # Python packages needed for building docs
   - autodocsumm>=0.2.2
+  - ipython
```

Comment:
This should probably also be added to `setup.py`?

Reply:
BTW, why do we need `ipython`? Wouldn't `ipykernel` be the correct package to run notebooks?

Reply:
It's for building the documentation. Without ipython the jupyter notebooks cannot be converted to .rst files.

Reply:
For running the notebooks I would install jupyterlab, but that is not needed to just build the documentation.